architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
+s390x default,alpine,onbuild,slim,stretch
```

Comment:
This line unfortunately can't have `alpine` in it yet: https://github.com/gliderlabs/docker-alpine/issues/304